tarball/tarball_reference.go

```diff
@@ -0,0 +1,88 @@
+package tarball
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/image/docker/reference"
+	"github.com/containers/image/image"
+	"github.com/containers/image/types"
+
+	imgspecv1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+// ConfigUpdater is an interface that references for "tarball" images also
```

Comment:
Finding this line a bit confusing.  Perhaps "references for "tarball" images" to "references for "tarball" that images"  or perhaps "references a "tarball" that images"

Reply:
Or after going a bit deeper here, perhaps "ConfigUpdater is an interface implemented for images that references "tarball:" images."

Reply:
Perhaps `ConfigUpdater is an interface that ImageReferences for "tarball" images also implement.`?

Reply:
sold!